Lib/datetime.py

```diff
@@ -1210,6 +1210,36 @@ def __reduce__(self):
         else:
             return (self.__class__, args, state)
 
+
+class IsoCalendarDate(tuple):
+
```

Comment:
```suggestion
__slots__ = ()
```

Reply:
@pppery Can you explain the reason about this suggestion?